doc/html/index.html

```diff
@@ -140,10 +140,14 @@ <h3>
 <li class="listitem">
```

Comment:
Please, remove changes to this file. 
Unfortunately, it gets modified during docs build, see #674, but the changes should be `git restore`-ed before committing.